Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:7.2-cli
+FROM php:8.1.9-cli
```

Comment:
you should change the minimum version of PHP to `8.0` and use `8.0` in the Dockerfile, this way when you create a new github tag a user can easily go from one PHP version to another